salt/utils/dns.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+import itertools
```

Comment:
This needs a `  # pylint: disable=E0403` after it to disable this pylint check, please.